cranelift/codegen/src/isa/mod.rs

```diff
@@ -176,7 +176,7 @@ pub type Legalize =
 
 /// This struct provides information that a frontend may need to know about a target to
 /// produce Cranelift IR for the target.
-#[derive(Clone, Copy)]
+#[derive(Clone, Copy, Hash)]
```

Comment:
I'm assuming this was used by a `PrimaryMap` in `compile` but I didn't look closely enough; just checking that it is used somewhere...

Reply:
Ah yeah the `Hash` here is used as a hash key for cached module compilations, so it is indeed used :)